bot_python_sdk/bluetooth_service.py

```diff
@@ -0,0 +1,60 @@
+import time
+import bluetooth
+from bluetooth.ble import DiscoveryService, BeaconService
+from bot_python_sdk.logger import Logger
+
+LOCATION = 'Bluetooth Service'
+
+
+
+class BluetoothService:
+
+    def discover_bluetooth_test(self):
+        try:
+            nearby_devices = bluetooth.discover_devices(lookup_names=True)
+            print("found %d devices" % len(nearby_devices))
+
+            for addr, name in nearby_devices:
+                print("  %s - %s" % (addr, name))
+            discover_response = {"message" : "success"}
+        except Exception as exception:
+            Logger.error(LOCATION, 'Failed discover_bluetooth_test')
+            discover_response = {"message" : "failed"}
+            raise exception        
+        return discover_response
+
+    def discover_ble_test(self):
```

Comment:
The SDK shouldn't discover devices.

Reply:
I have removed the method in latest commit.